python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -19,9 +19,12 @@
 import unittest
 import tempfile
 
+import pandas
```

Comment:
Hm .. we gonna fix this or do something. pandas isn't a required library for SQL package. Should probably skip this tests when pandas is not installed for now until we have a clear way to handle this. (see `pyspark.testing.sqlutils.have_pandas` and `pyspark.sql.tests.test_arrow_map`

Reply:
Interestingly, nothing in Spark Connect will work atm without pandas because we always call `toPandas` in the collection of the result. Let me know what you want to do.